services/workflow/workflow-manager/src/main/java/com/milesight/beaveriot/rule/manager/model/response/WorkflowComponent.java

```diff
@@ -0,0 +1,9 @@
+package com.milesight.beaveriot.rule.manager.model.response;
+
+import lombok.Data;
+
+@Data
+public class WorkflowComponent {
```

Comment:
There is already a BaseDefinition class that defines the node name and title

Reply:
Replaced with `BaseDefinition` and deleted `WorkflowComponent` now.